src/extension.ts

```diff
@@ -127,9 +126,13 @@ export function activate(context: vscode.ExtensionContext) {
             // prevent further popups.
             notifiedFiles.add(notification.origin());
           }
+          if (notification.isDone()) {
+          record(TelemetryActions.componentAnalysisTriggered, {depFileName: path.basename(vscode.window.activeTextEditor.document.fileName)});

```

Comment:
use notification.origin(), this will give you the right file.
```suggestion
          record(TelemetryActions.componentAnalysisTriggered, {filename: path.basename(notification.origin())});

```